packages/components/menu/src/menu.tsx

```diff
@@ -205,3 +204,6 @@ export const Menu: FC<MenuProps> = (props) => {
     </DescendantsContextProvider>
   )
 }
+
+Menu.displayName = "Menu"
+Menu.__ui__ = "Menu"
```

Comment:
There’s no need to specify displayName for components that don't use forwardRef.